pkg/nuctl/test/get_test.go

```diff
@@ -74,29 +78,55 @@ func (suite *GetTestSuite) TestGet() {
 
 		suite.Require().NoError(err)
 
-		// cleanup
+		// cleanup - try delete
+		// make sure to clean up after the test
+		// use nutctl to delete the function when we're done
```

Comment:
This comment was duplicated twice but it doesn't make sense in all three places - please read the test code when you're done writing it and think if it's still readable and if the comments make sense. Don't forget to add or update comments depending on what you changed.